gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftBuildTaskTest.java

```diff
@@ -41,12 +41,12 @@
   public TaskEnvironment taskEnvironment = new TaskEnvironment();
 
   private MockedConstruction<OpenshiftBuildService> openshiftBuildServiceMockedConstruction;
+  private final OpenShiftExtension extension = new TestOpenShiftExtension();
```

Comment:
NOT OK, either revert this change, or initialize the variable in the setUp method.